src/System.Data.SqlClient/tests/ManualTests/SQL/TcpDefaultForAzureTest/TcpDefaultForAzureTest.cs

```diff
@@ -0,0 +1,58 @@
+﻿using System.Diagnostics;
+using Xunit;
+
+namespace System.Data.SqlClient.ManualTesting.Tests
+{
+    public static class TcpDefaultForAzureTest
```

Comment:
Can you add more tests checking for a few other scenarios
1. Azure endpoint with tcp: speficied.
2. Azure endpoint with Named Pipe specified as the protocol.
3. Non-Azure endpoint with a protocol specified.

I want to make sure that a few regression scenarios are covered w.r.t this feature.